sources/packages/backend/workflow/test/2021-2022/fulltime-assessment/fulltime-assessment-2021-2022.e2e-spec.ts

```diff
@@ -0,0 +1,123 @@
+import { OfferingIntensity } from "@sims/sims-db";
+import {
+  AssessmentConsolidatedData,
+  AssessmentModel,
+  CalculatedAssessmentModel,
+} from "../../models";
+import { ZBClient } from "zeebe-node";
+import { getFakeAssessmentConsolidatedData } from "../../utils";
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+
+describe(`E2E Test Workflow  fulltime-assessment-${PROGRAM_YEAR}`, () => {
+  let zeebeClientProvider: ZBClient;
+  beforeAll(async () => {
+    zeebeClientProvider = new ZBClient();
+  });
+
+  it("Should generate expected fulltime assessment values for a single and independent student.", async () => {
+    const assessmentConsolidatedData = getFakeAssessmentConsolidatedData(
+      OfferingIntensity.fullTime,
+      PROGRAM_YEAR,
+      "2022-02-01",
+      "2022-05-24",
+    );
+
+    const expectedAssessmentData: AssessmentModel = {
+      weeks: assessmentConsolidatedData.offeringWeeks,
+      tuitionCost:
+        assessmentConsolidatedData.offeringActualTuitionCosts +
+        assessmentConsolidatedData.offeringMandatoryFees,
+      childcareCost: 0,
+      livingAllowance: 1672,
+      totalAssessedCost: 27672,
+      totalFamilyIncome: 40000,
+      totalFederalAward: 5600,
+      otherAllowableCost: 0,
+      transportationCost: 0,
+      secondResidenceCost: 0,
+      totalAssessmentNeed: 55049.19807692308,
+      booksAndSuppliesCost: 5000,
+      totalProvincialAward: 11037.417692307692,
+      alimonyOrChildSupport: 0,
+      federalAssessmentNeed: 26000,
+      exceptionalEducationCost:
+        assessmentConsolidatedData.offeringExceptionalExpenses,
+      provincialAssessmentNeed: 27593.544230769232,
+      parentAssessedContribution: null,
+      partnerAssessedContribution: null,
+      studentTotalFederalContribution: 216.34615384615384,
+      studentTotalProvincialContribution: 78.45576923076923,
+    } as AssessmentModel;
```

Comment:
Do we need to have the type explicitly defined at the variable declaration and also a cast at the end?